.mockery.yaml

```diff
@@ -0,0 +1,10 @@
+with-expecter: true
+filename: "{{.InterfaceName | snakecase}}_mock.go"
```

Comment:
nit: should we do `mock_{{.InterfaceName | snakecase}}.go` to match the actual mock name?